docs/performance-testing.md

```diff
@@ -1,36 +1,39 @@
 # Performance Testing
 
 TEAMMATES makes use of [JMeter](https://jmeter.apache.org/) for load and performance (L&P) testing, and uses the [JMeter Java API](https://jmeter.apache.org/api/index.html) and TestNG to automate the process of running performance tests.
-The performance test cases are located in the `teammates.e2e.cases.lnp` package.
+
+The performance test cases are located in the [`teammates.e2e.cases.lnp`](https://github.com/TEAMMATES/teammates/tree/master/src/e2e/java/teammates/e2e/cases/lnp) package.
 
 ## Creating Performance Tests
 
-Each new test case must inherit the base L&P test class, and implement the methods required for generating the test data and the JMeter L&P test plan. 
+Each new test case must inherit the `BaseLNPTestCase` class, and implement the methods required for generating the test data and the JMeter L&P test plan. 
 The L&P test plans are created in Java using the JMeter API.
 
-When running the L&P test, an equivalent `.jmx` file can be generated from this test plan.
+The inherited test cases can run JMeter test by calling `runJmeter` method. When passing the parameter `shouldCreateJmxFile` as `true`, an equivalent `.jmx` file can be generated from this test plan.
+
 To help with debugging, you can open this `.jmx` file in the JMeter GUI and add Listeners.
 
 To see a sample implementation of a test case, you can refer to `StudentProfileLNPTest`. It is a _simple_ test case which load tests a GET endpoint (`/webapi/student/profile`).
 
 ## Running Performance Tests
 
 If you want to use your own copy of [JMeter](https://jmeter.apache.org/download_jmeter.cgi), update the `test.jmeter.*` properties in `src/e2e/resources/test.properties` accordingly.  
-Start the backend server, i.e. `localhost:8080`, before running the performance tests.
+
+First, open the terminal and navigate to the root of project folder. Start the backend server, i.e. `./gradlew appengineRun`, before running the performance tests.
```

Comment:
Might just want to link them to the dev guide([`managing-the-dev-server-back-end`](https://github.com/TEAMMATES/teammates/blob/master/docs/development.md#managing-the-dev-server-back-end)) for this, since the command is different for windows